### DISCUSSION TITLE
LLaMa is kinda insane

### DISCUSSION THREAD

**Discussion by Mr-NI** [ggml-org/llama.cpp#215] [Show and tell]
<img width="549" alt="Screenshot 2023-03-16 at 5 43 53 PM" src="https://user-images.githubusercontent.com/69180073/225767750-6491105b-c8f0-4c34-a013-ce3c1819933f.png">
is this normal?

**Comment by Green-Sky** [comment]
Yes, llama was not finetuned and contains all kinds of hostile and nsfw content scraped from the web.
It also depends on what your initial prompt was.

**Reply by iHaagcom** [reply]
Can we fine tune it? Is so how?

**Reply by Green-Sky** [reply]
> how?

this is an example: https://github.com/tatsu-lab/stanford_alpaca

**Comment by slaren** [comment]
You can run a fine tuned  7B model with llama.cpp already using https://github.com/tloen/alpaca-lora. Follow the install instructions and run export_state_dict_checkpoint.py, you will get a .pth that can be converted and quantized with llama.cpp.
